scripts/analysis/call_expressions/python/README.md

```diff
@@ -0,0 +1,11 @@
+# Python call expressions preprocessing
+Analysis python call expressions, which include fully-qualified names of call expressions usage statistics.
+
+## Python call expressions analysis
+[call_expressions_analysis.py](call_expressions_analysis.py) will allow you to count the number of unique fq names in 
+python projects, keeping their category. 
+
+### Usage
+```bash
+python3 call_expressions_analysis.py --input /path/to/file/with/gathered/stats.csv --output /path/to/file/with/grouped/stats.csv
```

Comment:
please, use `-m` notation if you use `.py` running it can throw some exceptions with local imports